includes/batches/class-batch-comments.php

```diff
@@ -0,0 +1,73 @@
+<?php
+/**
+ * Comments batch class.
+ *
+ * @package Locomotive/Batch
+ */
+
+namespace Rkv\Locomotive\Batches;
+
+use WP_Comment_Query;
+use Rkv\Locomotive\Abstracts\Batch;
+
+/**
+ * Batch Comments class.
+ */
+class Comments extends Batch {
+	/**
+	 * The individual batch's parameter for specifying the amount of results to return.
+	 *
+	 * @var string
+	 */
+	public $per_batch_param = 'number';
+
+	/**
+	 * Default args for the query.
+	 *
+	 * @var array
+	 */
+	public $default_args = array(
+		'number' => 10,
+		'offset' => 0,
+		'no_found_rows' => false,
```

Comment:
If `no_found_rows` is set to true, which is the default, then `$query->found_comments` will always return `0`. The only way I can think of to get the total number of results (similar to `found_posts`) in comments is to set that to false, even though it slows down the query a bit.